server/script/server.sh

```diff
@@ -80,24 +79,28 @@ if [ -f "$ORIENTDB_PID" ]; then
     rm "$ORIENTDB_PID"
 fi
 
-# TO DEBUG ORIENTDB SERVER RUN IT WITH THESE OPTIONS:
-# -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=1044
-# AND ATTACH TO THE CURRENT HOST, PORT 1044
+# DEBUG OPTS, SIMPLY USE 'server.sh debug'
+DEBUG_OPTS=""
+if [ -n $1 ] && [ $1 = "debug" ]; then
+    DEBUG_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=1044"
+fi
 
 # ORIENTDB memory options, default to 512 of heap.
-
 if [ -z "$ORIENTDB_OPTS_MEMORY" ] ; then
     ORIENTDB_OPTS_MEMORY="-Xms512m -Xmx512m"
 fi
-# ORIENTDB MAXIMUM DISKCACHE IN MB, EXAMPLE, ENTER -Dstorage.diskCache.bufferSize=8192 FOR 8GB
-MAXDISKCACHE=""
+
+# ORIENTDB SETTINGS LIKE DISKCACHE, ETC
+if [ -z "$ORIENTDB_SETTINGS" ]; then
+    ORIENTDB_SETTINGS=""
```

Comment:
If `ORIENTDB_SETTINGS` is blank, set it to blank?

Am I missing something here or is this just redundant?